changelog/seqera-enterprise/v24.2.mdx

```diff
@@ -0,0 +1,132 @@
+- Update `lastUsed` on compute environment when creating job for Data Studio session.
+- Enable task `workDir` navigation using Data Explorer in the personal workspace.
+- Allow changing the `workDir` when launching a pipeline.
+- File name is not encoded when requesting download URL.
+- Accessing reports of Nextflow CLI runs fails with `TOWER_CONTENT_URL` set.
+- NF schema nested objects values are not preserved after going through input form. Removes processing of unsupported JSON schema types inside our pipeline input form components.
+- Show CORS modal when detecting incorrect eTag configuration for Data Explorer and update link to Seqera docs.
+
+## Breaking changes and warnings
+
+### Redisson properties deprecated
+
+From this version of Seqera Platform, `redisson.*` properties will be deprecated. If you have set `redisson.*` properties directly, do the following:
+
+- Replace `/redisson/*` references for the AWS parameter store with `TOWER_REDIS_*`.
+- Replace `redisson.*` references for the `tower.yml` with `TOWER_REDIS_*`. You can set `TOWER_REDIS_*` values directly in the YAML file (for example, `TOWER_REDIS_URL: redis://...`) in `tower.yml`.
+
+## Upgrade steps 
+
+1. This version includes an update to the Platform Enterprise H8 cache. **Do not start the upgrade while any pipelines are in a `running` state as active run data may be lost.**
+1. This version requires a database schema update. Make a backup of your Platform database prior to upgrade.
+1. If you are upgrading from a version older than 23.4.1, update your installation to version [23.4.4](https://docs.seqera.io/changelog/seqera-enterprise/v23.4_major) first, before updating to 24.2 with the steps below.
+1. For recommended Platform memory settings, add the following environment variable to your Platform configuration values (`tower.env`, `configmap.yml`, etc.):
+    ```bash 
+    JAVA_OPTS: -Xms1000M -Xmx2000M -XX:MaxDirectMemorySize=800m -Dio.netty.maxDirectMemory=0 -Djdk.nio.maxCachedBufferSize=262144
+    ```
+1. See [Upgrade installation](https://docs.seqera.io/platform/24.2/enterprise/upgrade) for database backup and installation upgrade guidance.
+
+:::info
+Docker Compose deployments require downtime while upgrading services. Restarting the application may take several minutes. See [Docker compose deployment](https://docs.seqera.io/platform/24.2/enterprise/docker-compose) for more information.
+
+For Kubernetes deployments, apply the 24.1 `tower-cron.yml` to your cron pod and wait for the cron pod to be running before applying the `tower-svc.yml` to your backend pod and restarting the service. If the cron pod update is interrupted, you may need to restore the instance from your DB backup and start again. See [Kubernetes deployment](https://docs.seqera.io/platform/24.2/enterprise/kubernetes) for more information. 
+
+For custom deployments with third-party services such as ArgoCD, [contact support](https://support.seqera.io) for assistance during upgrade. 
```

Comment:
Does Support know they are supporting ArgoCD-managed deployments?

Reply:
This verbiage was recommended in 24.1 upgrade steps by @swampie

Reply:
<img src="https://media1.giphy.com/media/FPjbHO0jJxGsE/giphy.gif?cid=6c09b952dtm7usdqkp4klhhmht2phfjcynjvwrldl9nbttae&ep=v1_gifs_search&rid=giphy.gif&ct=g"></img>

Reply:
we could remove the wording ArgoCd but it was just an example: we already have customers using ArgoCD and it's used internally by the company so I don't see any problem in using that.